### PR TITLE
Confirmation window for project deletion

### PR DESCRIPTION
**Pull request by Scalsol**

### CODE REVIEW

**Comment by fyu** [comment]
Can you show a screenshot?

**Comment by Scalsol** [comment]
Oh ok. When press the X icon, this confirmation window will show up.
图片

**Comment by Scalsol** [comment]
图片

File: app/src/components/sidebar_projects.tsx [hunk 1]
```diff
@@ -74,7 +74,11 @@ class ProjectList extends React.Component<ProjectListProps, ProjectListState> {
               <CloseIcon
                 titleAccess="Delete project"
                 fontSize="small"
-                onClick={() => this.deleteProject(project)}
+                // onClick={() => this.deleteProject(project)}
```
Comment (fyu) [inline app/src/components/sidebar_projects.tsx:77]:
```suggestion
```

**Review by fyu — APPROVED** [review]

**Pull request merged**